src/Vault/Vault.csproj

```diff
@@ -10,19 +10,19 @@
     <Company>Hashicorp</Company>
     <AssemblyTitle>Vault OpenAPI Library</AssemblyTitle>
     <Description>A Vault Client library generated from an OpenAPI doc</Description>
-    <Copyright>No Copyright</Copyright>
     <RootNamespace>Vault</RootNamespace>
     <Version>0.1.0-beta</Version>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Vault.xml</DocumentationFile>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageLicenseExpression>MPL-2.0</PackageLicenseExpression>
     <RepositoryUrl>https://github.com/hashicorp/vault-client-dotnet.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <PackageReleaseNotes>Minor update</PackageReleaseNotes>
```

Comment:
all the other xml names with file paths have the word "File" at the end of it (eg: "DocumentationFile", "PackageReadmeFile") does the build tool know to switch from raw strings vs file path?

Reply:
good call. Lemme double check this. I just packed the nuget file to test. But yeah, I'll see if let's you pipe in a release notes